app/src/test/java/com/example/giftly/Font_SizeTests.java

```diff
@@ -0,0 +1,135 @@
+package com.example.giftly;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.robolectric.Shadows.shadowOf;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.SeekBar;
+import android.widget.TextView;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.RuntimeEnvironment;
+
+@RunWith(RobolectricTestRunner.class)
+public class Font_SizeTests {
+    private FontSizeScreen activity;
+
+    @Before
+    public void setUp() {
+        System.out.println("Setting up test...");
+        activity = Robolectric.buildActivity(FontSizeScreen.class).create().visible().get();
+        System.out.println("Setup complete");
+    }
+
+    //Testing on whether the user is able to go into the font size setting
+    @Test
+    public void testOnCreate() {
```

Comment:
Use more descriptive test method names; testOnCreate is generic

Reply:
renamed the test method name to testAccessFontSizeFeature()